src/list-data/request-actions.ts

```diff
@@ -1,63 +1,78 @@
-import { RequestCode } from '@/enums'
+import { NrRequestActionCodes } from '@/enums'
 import { RequestActionsI } from '@/interfaces/models'
 
+/** All the request actions - some are headers and others are selectable items. */
 export const RequestActions: RequestActionsI[] = [
   {
-    text: 'Register or Incorporate a',
+    isHeader: true,
+    group: 0,
+    text: 'For businesses that do not exist yet:'
+  },
+  {
+    group: 0,
+    text: 'Start a new BC-based business',
+    subtext: 'Incorporation or Registration',
     shortDesc: 'New Request',
-    value: RequestCode.NEW,
-    blurbs: `Create a new business in British Columbia or register a business you formed in another province or
-              territory, country or federal jurisdiction so that you may also conduct business here in BC.`
+    value: NrRequestActionCodes.NEW_BUSINESS
   },
   {
-    text: 'Relocate into',
-    shortDesc: 'Move Request',
-    value: RequestCode.MVE,
-    blurbs: `Transfer a corporation you formed in another jurisdiction so that it becomes a BC company.`
+    isHeader: true,
+    group: 1,
+    text: 'For businesses that already exist outside BC:'
   },
-  // COMMENTED OUT FOR FUTURE IMPLEMENTATION
-  // {
-  //   text: 'Assume a New Name in BC',
-  //   shortDesc: 'Assumed Name Request',
-  //   value: RequestCode.ASSUMED,
-  //   blurb: `You have an existing business in another province. You are closing your business there and moving your
-  //           business to BC, however, the name of your business is already in use in BC`
-  // },
```

Comment:
This is an obsolete primary action, but it is dynamically added as a Request Type Option if the Request Type Code is AS/AL/XASO/XCASO/UA.

This is existing code but I'll try to untangle it a bit more as part of this ticket (possibly in a future PR). (See also Name Type enum above.)